src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+
+interface ConfirmationBoxProps {
+  open: boolean;
+}
+
+export const ConfirmationBox = styled.div<ConfirmationBoxProps>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 2rem 3rem;
+  position: fixed;
+  background-color: ${({ theme }) => theme.colors.background};
+  width: 100%;
+  left: 0;
+  ${(props) =>
```

Comment:
```suggestion
  position: fixed;
  left: 0;
  background-color: ${({ theme }) => theme.colors.background};
  width: 100%;

  ${(props) =>
```

It is a good practice to put the positioning next to the `position: fixed`.

Reply:
OK fixed 868fb95